### DISCUSSION TITLE
Help with webpanel

### DISCUSSION THREAD

**Discussion by johnhiltonza** [jekhokie/raspberry-noaa-v2#322] [Q&A] [answered]
Hi all,

Setup my first Raspberry Pi to try and get this app to work... struggling with making the webpanel available on my local network. Have used the IP of the host (192.168.1.81) in the config file, done the setup, I can browse to the IP when working directly off the raspberry Pi via its graphical interface, but when trying to access from another PC at the same IP, I do not get the webpanel... what am I missing?

Thanks for the help...

**Comment by jekhokie** [comment, marked as answer]
@johnhiltonza this might be kind of difficult to troubleshoot over GitHub - care to join us on Discord so we can help you out? Here's a link: https://discord.gg/368C6PuC

Couple things pop to mind:
1. Any firewalls on your local network that could be blocking access to the Pi instance?
2. Any firewalls running on the Pi that could be blocking external access?
3. Are the devices on your home network in the same IP subnet as the Pi device (if so, can you at least ping the IP address from another device on your local network)?

**Comment by johnhiltonza** [comment]
Thanks Justin,

Joined the Discord server now, I realize now I need to use https:// for the
address. Then it worked, but not I seem to have an issued with it not
interfacing with my Nooelec NESDR SMArTee v2... but will work it out, I am
"OK" with command line type stuff, but no expert with the OS, but will
learn here slowly!

Doing a clean OS install now and will start fresh.

Catch you on Discord.

Best,
John
